app/grandchallenge/components/templates/components/civ_set_list.html

```diff
@@ -39,6 +39,10 @@ <h2>{{ base_object.title }} {{ base_object.civ_set_model|verbose_name_plural|tit
                 hx-push-url="{{ base_object.bulk_delete_url }}"
                 hx-vals='{"delete-all": "True"}'
                 hx-target="body"
+                {% if delete_all_enabled is not True %}

```

Comment:
`delete_all_enabled` does not exist in the context for non-reader studies...

Reply:
For any of the usages of this file. This PR would again break existing implementations. Again, test everything that you touch.

Reply:
Done